src/disco/gui/fd_gui.c

```diff
@@ -1247,8 +1248,9 @@ fd_gui_handle_completed_slot( fd_gui_t * gui,
   ulong failed_txn_count = msg[ 3 ];
   ulong nonvote_failed_txn_count = msg[ 4 ];
   ulong compute_units = msg[ 5 ];
-  ulong fees = msg[ 6 ];
-  ulong _parent_slot = msg[ 7 ];
+  ulong transaction_fee = msg[ 6 ];
+  ulong priority_fee = msg[ 7 ];
```

Comment:
mcache MTU needs to be increased in the topology for this link

Reply:
Good point.  It looks like the slot reset message is also sent over that link, and so it's been sized large enough to cover the chain of parents for a slot reset, and as a result also large enough for slot complete?

Reply:
Ah ok!